crates/gateway-types/src/reply.rs

```diff
@@ -856,8 +856,7 @@ pub mod transaction {
             Self {
                 l1_gas: value.l1_gas.into(),
                 l2_gas: value.l2_gas.into(),
-                // TODO: add this when adding support for Starknet 0.13.4
```

Comment:
BTW: are there more of these 0.13.4 related TODOs?

Reply:
not in this file... :-) I'll look elsewhere as well...